tests/chocolatey-tests/choco-install.Tests.ps1

```diff
@@ -1392,4 +1392,36 @@ Describe "choco install" -Tag Chocolatey, InstallCommand {
             $result2.Lines | Should -Contain "Circular dependency detected 'circulardependency2 0.0.1 => circulardependency1 0.0.1 => circulardependency2 0.0.1'."
         }
     }
+
+    Context "Installing package that extracts local zip archive while disabling logging" {
+        BeforeAll {
+            Restore-ChocolateyInstallSnapshot
+
+            $Output = Invoke-Choco install zip-log-disable-test --verbose --debug -y
+        }
+
+        It "Exits with Success (0)" {
+            $Output.ExitCode | Should -Be 0
+        }
+
+        It "Does not output extracted file path '<_>'" -ForEach @('tools\'; 'tools\chocolateybeforemodify.ps1'; 'tools\chocolateyinstall.ps1'; 'tools\chocolateyuninstall.ps1'; 'zip-log-disable-test.nuspec') {
+            $Output.String | Should -Not -Match "- $([regex]::Escape($_))"
+        }
+    }
+
+    Context "Installing package that extracts external zip archive while disabling logging" -Tag Internal {
+        BeforeAll {
+            Restore-ChocolateyInstallSnapshot
+
+            $Output = Invoke-Choco install zip-log-disable-test-external --verbose --debug -y
+        }
+
+        It "Exits with Success (0)" {
+            $Output.ExitCode | Should -Be 0
+        }
+
+        It "Does not output extracted file path '<_>'" -ForEach @('tools\'; 'tools\chocolateybeforemodify.ps1'; 'tools\chocolateyinstall.ps1'; 'tools\chocolateyuninstall.ps1'; 'zip-log-disable-test.nuspec') {

```

Comment:
```suggestion
        It "Does not output extracted file path '<_>'" -ForEach @('tools\'; 'tools\chocolateybeforemodify.ps1'; 'tools\chocolateyinstall.ps1'; 'tools\chocolateyuninstall.ps1'; 'zip-log-disable-test-external.nuspec') {

```

I imagine we want to test that the nuspec for the package being extract isn't included in the output 🤷

Reply:
Nope, the archive have the nuspec file `zip-log-disable-test.nuspec`, this is what we are testing against.

The output doesn't actually report the path it gets extracted to, but only the paths inside of the archive.

Reply:
Oh, and it's the same zip which is just the package. That makes sense. Ok, then I have a different suggestion that will reduce duplication (I think, it might need some testing...)